types/xmlbuilder/index.d.ts

```diff
@@ -67,7 +67,7 @@ declare namespace xmlbuilder {
         importDocument(input: XMLElementOrXMLNode): XMLElementOrXMLNode;
         root(): XMLElementOrXMLNode;
         document(): any;
-        end(options?: Object): string;
+        end(writerOrOptions?: XMLWriterOrXMLToStringOptions): string;
```

Comment:
The names make me think a union type might have been intended (i.e. do you expect either options or a writer or is it reasonable to specify both?).

Reply:
Latter. It is reasonable for the method to specify both 'XMLWriter' and 'XMLToStringOptions'. But as you thought, my changes did not express it properly.
I changed the name of type. Thank you for your review.

Reply:
Latter. It is reasonable for the method to specify both 'XMLWriter' and 'XMLToStringOptions'. But as you thought, my changes did not express it properly.
I changed the name of type. Thank you for your review.